Program/commandline.h

```diff
@@ -105,7 +111,10 @@ class CommandLine
 		std::cout << "[-lambda <int>] Number of solutions created before reaching the maximum population size (i.e., generation size). Defaults to 40 " << std::endl;
 		std::cout << "[-nbElite <int>] Number of elite individuals. Defaults to 5                                                                     " << std::endl;
 		std::cout << "[-nbClose <int>] Number of closest solutions/individuals considered when calculating diversity contribution. Defaults to 4      " << std::endl;
-		std::cout << "[-targetFeasible <double>] target ratio of feasible individuals in the last 100 generatied individuals. Defaults to 0.2         " << std::endl;
+		std::cout << "[-nbIterPenaltyManagement <int>] Number of iterations between penalty updates. Defaults to 100                                  " << std::endl;
```

Comment:
Now that 100 is a parameter should we put a different wording?

Reply:
Do you have a suggested alternative? I have already rephrased `targetFeasible`, `penaltyIncrease`, and `penaltyDecrease`, but I'm happy to change them!